CopsAndRobbers/Assets/FindControls.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Me.DerangedSenators.CopsAndRobbers
+{
+    /// @authors Nisath Mohamed Nasar
+    /// <summary>
+    /// Find appropriate Platform and display controls accordingly.
+    /// </summary>
+    public class FindControls : MonoBehaviour
+    {
+        public GameObject mobileControls, windowsControls;
+        
+#if UNITY_STANDALONE || UNITY_WEBPLAYER
+        private void EnableControls()
```

Comment:
You could have all controls set to false and then only toggle one to true.
Also change name from windowsControls to desktopControls

Reply:
fixed this can you recheck